hack/build-antrea-linux-all.sh

```diff
@@ -140,9 +145,11 @@ if $PULL; then
         docker pull $PLATFORM_ARG ubuntu:24.04
         docker pull $PLATFORM_ARG golang:$GO_VERSION
     else
-        docker pull ${DOCKER_REGISTRY}/antrea/ubuntu:24.04
-        docker tag ${DOCKER_REGISTRY}/antrea/ubuntu:24.04 ubuntu:24.04
-        docker pull ${DOCKER_REGISTRY}/antrea/golang:$GO_VERSION
+        if [[ $TESTBED_TYPE != "kind" && $TESTBED_TYPE != "kind-flexible-ipam" ]]; then
+            docker pull ${DOCKER_REGISTRY}/antrea/ubuntu:22.04
+            docker pull ${DOCKER_REGISTRY}/antrea/golang:$GO_VERSION 
+        fi
+        docker tag ${DOCKER_REGISTRY}/antrea/ubuntu:22.04 ubuntu:22.04
```

Comment:
that doesn't seem necessary, but also the change is incorrect as it is reverting from ubuntu:22.04 to ubuntu:24.04

if you don't want to pull base images for some reason, do not specify `--pull`